ui/apps/everest/.e2e/pr/db-cluster-details/edit-db-cluster/resources-step.e2e.ts

```diff
@@ -0,0 +1,231 @@
+// everest
+// Copyright (C) 2023 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import { expect, Page, test } from '@playwright/test';
+import { getTokenFromLocalStorage } from '@e2e/utils/localStorage';
+import {
+  deleteDbCluster,
+  findDbAndClickRow,
+} from '@e2e/utils/db-clusters-list';
+import { getClusterDetailedInfo } from '@e2e/utils/storage-class';
+import {
+  moveForward,
+  populateBasicInformation,
```

Comment:
this is unused

Reply:
fixed both in https://github.com/percona/everest/pull/785/commits/d841c2f1a0629eee4702b5c50a7ac10a05c16076